go.mod

```diff
@@ -22,8 +22,8 @@ require (
 
 require (
 	github.com/graph-gophers/graphql-go v1.5.1-0.20230110080634-edea822f558a
-	github.com/mattermost/mattermost/server/public v0.0.6-0.20230705221755-44a95d9f6901
-	github.com/mattermost/mattermost/server/v8 v8.0.0-20230705221755-44a95d9f6901
+	github.com/mattermost/mattermost/server/public v0.0.9-0.20230822143856-e48efdc5daae
+	github.com/mattermost/mattermost/server/v8 v8.0.0-20230822143856-e48efdc5daae
```

Comment:
Pointed these at Master for now. Would a tag be better?

Reply:
This points at a specific commit so essentially the same as a tag, just less human readable.